src/actions/actionsFilter.js

```diff
@@ -0,0 +1,4 @@
+import { CHECK_CHECKBOX, NULL_STATE_ON_PAGE_CHANGE } from '../constants/actionTypes';
+
+export const checkCheckbox = (category, id, clicked) => ({ type: CHECK_CHECKBOX, payload: { category, id, clicked } });
+export const nullState = () => ({ type: NULL_STATE_ON_PAGE_CHANGE });
```

Comment:
purpose?

Reply:
I haven't finished it yet, but it's for deleting everything from store in order to not show products which are connected to the diffrenct page